packages/edge-login-ui-rn/src/common/actions/LoginAction.js

```diff
@@ -326,24 +310,11 @@ export function userLogin (data: Object, backupKey?: string) {
 export function getEdgeLoginQrCode () {
   return async (dispatch: Dispatch, getState: GetState, imports: Imports) => {
     const context = imports.context
-    const callback = imports.callback
     const myAccountOptions = {
       ...imports.accountOptions,
-      callbacks: {
-        ...imports.accountOptions.callbacks,
-        onLoggedOut: () => {
-          dispatch(dispatchAction(Constants.RESET_APP))
-        }
-      },
       displayImageUrl:
         'https://github.com/Airbitz/edge-brand-guide/blob/master/Logo/Mark/Edge-Final-Logo_Mark-Green.png',
-      displayName: 'Edge Wallet',
-      onProcessLogin: username => {
-        // throw spinner
-      },
-      onLogin: (e, account) => {
```

Comment:
We need to do an `EdgeContext.on('login')` here to replace the `onLogin` callback.